edwardpizarro-fatima.js

```diff
@@ -0,0 +1,222 @@
+//Question 1
+//Write a function calculateCube that takes a single number and prints the volume of a cube made from that number.
+//console.log(calculateCube(5));
+//=> 125
+
+function calculateCube(){
+    console.log(5*25)
+  }
+   calculateCube()
+  //I wasnt sure about this solution, but it came out to 125. Im pretty sure there is a right way of doing this just not sure what it is.
+  
+  
+  //Question 2
+  //Is a Vowel?
+  // Write a function isAVowel that takes a character (i.e. a string of length 1) and returns true if it is a vowel, false otherwise. The vowel could be upper or lower case.
+  
+  let vowels = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U']
+  const isAVowel = (letter) => {
+    if (vowels.indexOf(letter.toUpperCase()) !== -1) {
+      return true
+    }
+    return false
+  }
+  console.log(isAVowel('a'))
+  //Got some help from Seth and Trez on this one. On a scale from 1-10 id say im at a 5 with this one for i dont 100% understand it.
+  
+  
+  
+  //Question 3
+  // Write a function getTwoLengths that accepts two parameters (strings). The function should return an array of numbers where each number is the length of the corresponding string.
+  
+  // console.log(getTwoLengths("Hank", "Hippopopalous"));
+  // => [4, 13]
+  
+  const getTwoLengths = (string1, string2) => {
+    let arr = [string1.split('').length, string2.split('').length]
+    return arr
+    //console.log(string1.length, string2.length)
+  }
+  // getTwoLengths('Name something else', 'Names')
+  //Me Seth and Trez were working on this together but couldnt get past putting one parameter as an argument. I wasnt sure on how to do that but have decent understanding on whats going on. 
```

Comment:
yeah, let's talk about this! it's really important you have a solid grasp on these concepts.